src/test/java/core/basesyntax/service/RegistrationServiceImplTest.java

```diff
@@ -0,0 +1,132 @@
+package core.basesyntax.service;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+import static org.junit.jupiter.api.Assertions.assertThrows;
+
+import core.basesyntax.dao.StorageDaoImpl;
+import core.basesyntax.db.Storage;
+import core.basesyntax.model.User;
+import org.junit.jupiter.api.AfterEach;
+import org.junit.jupiter.api.BeforeAll;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+
+class RegistrationServiceImplTest {
+    private static RegistrationServiceImpl registrationService;
+    private static User user;
+    private static StorageDaoImpl storageDao;
+
+    @BeforeAll
+    static void beforeAll() {
+        registrationService = new RegistrationServiceImpl();
+        user = new User();
+        storageDao = new StorageDaoImpl();
+    }
+
+    @BeforeEach
+    void setUp() {
+        user.setAge(23);
+        user.setLogin("Login123");
+        user.setPassword("qwerty123");
+    }
+
+    @Test
+    void register_nullUser_NotOk() {
+        assertThrows(RuntimeException.class,
+                () -> registrationService.register(null));
+    }
+
+    @Test
+    void register_invalid_login_NotOk() {
```

Comment:
```suggestion
    void register_invalidLogin_NotOk() {
```
![image](https://github.com/mate-academy/jv-registration-validation/assets/20326858/b3b6dd6d-0120-4903-bb75-a266e0a0cb3f)